lib/preprocess.js

```diff
@@ -34,6 +34,21 @@ export const preprocess = async ({
   recordTelemetry,
   logger
 }) => {
+  if (!rounds[roundIndex]) {
+    rounds[roundIndex] = []
+    rounds[roundIndex]._strings = new Map()
```

Comment:
I don't want to keep all strings forever. By attaching the lookup table to the `rounds` structure, we will automatically delete it after the round is evaluated.

Adding new properties to a built-in Array instance is a bit hacky. A nicer option is to refactor our code so that `rounds` store an object `{measurements, strings}` instead.

Is it worth it?

Reply:
Let's have it like this, since it's techy itself